### DISCUSSION TITLE
Attaching particles to blocks

### DISCUSSION THREAD

**Discussion by Danispace** [Zrips/CMI#5470] [Feature Requests]
It would be good if you could attach particles to blocks and could do more with particles.

**Comment by CloudeLecaw** [comment]
What you mean by do more?

There is allready na nice way around to attach particles ton a block.

- Create a little portal over the block or area of your desire
- give it some particle effecsts
- Dont set an Output teleport.
-  set the execute command without target to false

voila..you have an particleeffect attached to a block. 
- Particleeffect and amount of particles can be choosen in the gui of that portal
